docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,387 @@
+4. In the group policy editor, select:
+
+```text
+Computer Configuration > Policies > Windows Settings > Security Settings > Public Key Policies
+```
+
+5. Right click on `Trusted Root Certification Authorities` and select `Import`.
+6. Click through the wizard, selecting your CA file (`db-ca.cer`).
+
+<Figure align="left" bordered caption="Import Teleport CA">
+  ![Import Teleport CA](../../../img/desktop-access/ca.png)
+</Figure>
+
+### Enable smart card service.
+
+Teleport performs certificate based authentication by emulating a smart card.
+
+1. Still editing your `Teleport DB Access`, select:
+
+```text
+Computer Configuration > Policies > Windows Settings > Security Settings > System Services
+```
+
+2. Double click on `Smart Card`, select `Define this policy setting` and switch
+   to `Automatic` then click `OK`.
+
+<Figure align="left" bordered caption="Enable the Smart Card Service">
+  ![Enable Smartcard](../../../img/desktop-access/smartcard.png)
+</Figure>
+
+<Admonition type="note" title="gpupdate.exe">
+  You will be modifying GPOs, and sometimes GPO modifications can take some time
+  to propagate to all hosts. You can force your changes to take effect
+  immediately on your current host at any time by opening a PowerShell prompt
+  and running `gpupdate.exe /force` (though their effects may still take time to
+  propagate to other machines on the domain).
+</Admonition>
+
+### Publish Teleport CA
+
+This step enables the domain controllers to trust the Teleport CA, which will
+allow smart card logons via Teleport to succeed.
+
+On a machine which is joined to your domain and logged in as an account in the
+`Domain Administrators` group, run the two commands below at a PowerShell prompt
+to publish the Teleport CA to your Active Directory domain (using the path
+to the exported Teleport `db-ca.cer` file that you copied above):
+
+```powershell
+certutil –dspublish –f <PathToCertFile.cer> RootCA
+certutil -dspublish -f <PathToCertFile.cert> NTAuthCA
+```
+
+### Publish Teleport CRL
+
+On the same machine, run the command below at a PowerShell promt to publish the
+Teleport CRL to your Active Directory domain (using the path to the exported
+`db-ca.crl` file that you copied above).
+
+```powershell
+certutil -dspublish -f <PathToCRLFile.crl> TeleportDB
+```
+
+<Admonition type="tip">
+  To avoid waiting until the certificate propagation, you can force the CA
+  retrieval from LDAP after importing the CA and CRL with the command:
+
+  ```powershell
+  certutil -pulse
+  ```
+</Admonition>
+
+## Step 3/7. Export LDAP CA certificate
+
+To authenticate users, Teleport uses LDAPS. This means that Teleport needs to
+know that the certificate sent by the server during the initial SSL connection
+is trusted. This is done by specifying the LDAP CA certificate on the database
+configuration, but first, we need to export it from your AD. It can be done by
+running the following PowerShell script on your Windows instance:
```

Comment:
- I think the "but first" in the middle of this paragraph makes it a bit harder to follow. Is there a way to describe the first step first?
- This paragraph should use [active voice](https://www.grammarly.com/blog/active-vs-passive-voice/) for clarity. Instead of "it can be done", for example, describe who/what is performing the action.